src/music_pre.c

```diff
@@ -31,8 +31,70 @@ struct unkStruct_3000FD8
 
 extern struct unkStruct_3000FD8 gUnknown_3000FD8[8];
 
+struct BGControlStruct
+{
+    u16 padding;
+    u16 unk2;
+    s16 hofs;
+    s16 vofs;
+};
```

Comment:
Since this is being duplicated, I think this should be moved to a new include header